hyperscale/001-db-user-creation.sql

```diff
@@ -0,0 +1,3 @@
+create user citus with encrypted password 'Citus123';
```

Comment:
```suggestion
CREATE USER citus WITH ENCRYPTED PASSWORD 'Citus123';
```

It is better to keep capital letters when using sql keywords. Please go over other `.sql` files for similar styles.

Reply:
Fixed